explorer/testdata/assert/assert.carbon

```diff
@@ -2,12 +2,15 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// This won't auto-update because it uses a regex for the prelude line number,
+// so that it doesn't need to be updated on every prelude change.
+//
 // RUN: %{not} %{explorer} %s 2>&1 | \
 // RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes=false %s
 // RUN: %{not} %{explorer} --parser_debug --trace_file=- %s 2>&1 | \
 // RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes %s
-// AUTOUPDATE: %{explorer} %s
-// CHECK: RUNTIME ERROR: {{.*}}/explorer/data/prelude.carbon:431: "HALLO WELT"
+// NOAUTOUPDATE
+// CHECK: RUNTIME ERROR: {{.*}}/explorer/data/prelude.carbon:{{.*}}: "HALLO WELT"
```

Comment:
Do we want autoupdate to always use `{{.*}}` instead of line numbers in any file in `/explorer/data/`?

Reply:
I think we shouldn't make decisions like that based on one example.

Reply:
Also, do note, I'm trying to fix the build -- we should probably prioritize a fix over debate about long-term behavior.